data_safe_haven/commands/admin.py

```diff
@@ -66,7 +149,27 @@ def remove_users(
         ),
     ],
 ) -> None:
-    admin_remove_users(usernames)
+    """Remove existing users from a deployed Data Safe Haven"""
+    context = ContextSettings.from_file().assert_context()
+    config = Config.from_remote(context)
+    pulumi_config = DSHPulumiConfig.from_remote(context)
+
+    shm_name = context.shm_name
```

Comment:
Is `shm_name` used?

Reply:
Yes, I think they all are.

Reply:
I don't think it's used anywhere in the remainder of this function.